dkg/paranet.py

```diff
@@ -259,8 +265,13 @@ def is_voter(
         self,
         ual: UAL,
         address: Address | None = None,
-        incentives_type: ParanetIncentivizationType = ParanetIncentivizationType.NEUROWEB,
```

Comment:
This is a breaking change, we shouldn't do that imo

Reply:
I c your logic, but I would rather make default None and if it's None, then figure it out from the ual